### PR TITLE
Add filter on ids for timeTrackings.list

### PR DESCRIPTION
**Pull request by JordyDeGroeve**
Ensure the [base branch](https://github.com/teamleadercrm/api/pull/680) is merged first

### CODE REVIEW

**Review by mrtus — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by mattkomarnicki — APPROVED** [review]

**Review by MathiasDL — APPROVED** [review]

**Comment by JordyDeGroeve** [comment]
⚠️  Let's wait until base branch is merged

**Pull request merged**
